service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/digitaltwin/models/DigitalTwinInvokeCommandRequestOptions.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.service.digitaltwin.models;
+
+/**
+ * General request options that are applicable, but optional, for invoke command APIs.
+ */
+public final class DigitalTwinInvokeCommandRequestOptions {
+
+    /**
+     * The time (in seconds) that the service waits for the device to come online.
+     * The default is 0 seconds (which means the device must already be online) and the maximum is 300 seconds.
+     */
+    private Integer connectTimeoutInSeconds;
+
+    /**
+     * The time (in seconds) that the service waits for the method invocation to return a response.
+     * The default is 30 seconds, minimum is 5 seconds, and maximum is 300 seconds.
+     */
+    private Integer responseTimeoutInSeconds;
+
+    public Integer getConnectTimeoutInSeconds()
+    {
+        return connectTimeoutInSeconds;
+    }
+
+    public void setConnectTimeoutInSeconds(Integer connectTimeoutInSeconds)
```

Comment:
javadoc missing

Reply:
this and below

Reply:
This is generated using delombok which does not add documentation unfortunately from private fields above

Reply:
I have added documentation for now